package/base-files/files/etc/init.d/led

```diff
@@ -49,6 +51,11 @@ load_led() {
 	[ -e /sys/class/leds/${sysfs}/brightness ] && {
 		echo "setting up led ${name}"
 
+
+		max_brightness="$(cat /sys/class/leds/${sysfs}/max_brightness)"
```

Comment:
probably nicer to avoid dropping to a sub-shell and just use 'read'?

Reply:
sure, I'll fix it, thanks!

Reply:
done